app/grants/tests/test_views.py

```diff
@@ -27,12 +27,14 @@ def test_fetching_grant_categories_from_api(self):
                 ['community',5],
                 ['eth2.0',6],
                 ['eth1.x',7],
-                ['twitter', 8],
-                ['reddit', 9],
-                ['blog', 10],
-                ['notes', 11],
-                ['COVID19 research',12],
-                ['COVID19 response',13]
+                ['devEx', 8],
```

Comment:
what do these numbers mean that are in this response? if they are meaningful, we should probably append the new categories to the end of this list, right?

Reply:
right!

Reply:
I will change!

Reply:
@danlipert   The API returns according to the categories, so DevEx and Usability is above twitter. The order is Technology, Community, Health